app/views/community/show.html.haml

```diff
@@ -10,7 +10,7 @@
             Accelerate your learning with Exercism&apos;s community.
           Ask questions, explore ideas, and get to know millions of other developers upskilling on Exercism.
 
-        = link_to 'https://forum.exercism.org/', class: "btn-primary btn-l mb-20 w-[fit-content]" do
+        = link_to 'https://forum.exercism.org/', target: "_blank", rel: "noreferrer", class: "btn-primary btn-l mb-20 w-[fit-content]" do
```

Comment:
I don't know if we are worried about omitting `Referer`_(sic!)_ header, but as a best practice I did it like this.